inference-engine/src/mkldnn_plugin/nodes/mkldnn_reorder_node.cpp

```diff
@@ -77,33 +127,8 @@ void MKLDNNReorderNode::createPrimitive() {
     if (getSelectedPrimitiveDescriptor() == nullptr)
         IE_THROW() << "Preferable primitive descriptor is not set.";
 
-    auto inDims = srcMemPtr->GetShape().getStaticDims();
-
-    if (!isOptimized) {
-        const auto &parentMem = getParentEdgeAt(0)->getMemory();
-        if (MKLDNNPlugin::one_of(inDims.size(), 4, 5) &&
-                inDims[1] <= 64 &&
-                inDims[1] >= 16 &&
-                parentMem.getDesc().hasLayoutType(LayoutType::nspc) &&
-                (parentMem.GetDescWithType<BlockedMemoryDesc>()->getPaddedElementsCount() / inDims[1]) >= 128 &&
-                getChildEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::ncsp) &&
-                parentMem.getDesc().getPrecision() == Precision::FP32 &&
-                getChildEdgeAt(0)->getMemory().getDesc().getPrecision() == Precision::FP32) {
-            // oneDNN JIT reorder shows bad perf for nspc to ncsp reorder case so we fallback on simple c++ implementation
-            canUseOptimizedNspc2Ncsp = true;
-        } else if (!impl::cpu::x64::mayiuse(impl::cpu::x64::avx2) &&
-                   MKLDNNPlugin::one_of(inDims.size(), 4, 5) &&
-                   parentMem.getDesc().hasLayoutType(LayoutType::ncsp) &&
-                   getChildEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::nspc) &&
-                   parentMem.GetDataType() == getChildEdgeAt(0)->getMemory().GetDataType() &&
-                   MKLDNNExtensionUtils::sizeOfDataType(parentMem.GetDataType()) == 1) {
-            // oneDNN doesn't provide JIT reorder impl for non-avx2 targets so we fallback on simple c++ implementation which shows better perf
-            canUseOptimizedNcsp2Nspc = true;
-        } else {
-            createReorderPrimitive(srcMemPtr->GetDescWithType<DnnlMemoryDesc>()->getDnnlDesc(), srcMemPtr->GetPrimitive().get_data_handle(),
-                                   dstMemPtr->GetDescWithType<DnnlMemoryDesc>()->getDnnlDesc(), dstMemPtr->GetPrimitive().get_data_handle());
-        }
-    }
+    createReorderPrimitive(srcMemPtr->GetDescWithType<DnnlMemoryDesc>()->getDnnlDesc(), srcMemPtr->GetPrimitive().get_data_handle(),
```

Comment:
in `createReorderPrimitive` we no need to recreation `src_blocked` and `src_blocked` move to prepare params